src/YesSql.Core/Session.cs

```diff
@@ -900,7 +908,15 @@ public async ValueTask DisposeAsync()
 
             _disposed = true;
 
-            await CommitOrRollbackTransactionAsync();
+            try
+            {
+                await CommitOrRollbackTransactionAsync();
+            }
+            catch
+            {
+                _transaction = null;
```

Comment:
I think this should go into a `finally` block for clean up

Reply:
No because the standard path does it if it doesn't fail